package.json

```diff
@@ -153,6 +154,7 @@
     "reflect-metadata": "^0.2.0",
     "rxjs": "^7.8.1",
     "short-unique-id": "^5.2.0",
+    "spellchecker-cli": "^7.0.0",
```

Comment:
We need to move this to a devDep so that it doesn't get included in Gateway production code.

Reply:
Doh, my mistake. Fixed.